data/helpers.d/psql

```diff
@@ -67,18 +67,18 @@ ynh_psql_execute_file_as_root() {
 #
 # [internal]
 #
-# usage: ynh_psql_create_db db [user [pwd]]
+# usage: ynh_psql_create_db db [user]
 # | arg: db - the database name to create
 # | arg: user - the user to grant privilegies
-# | arg: pwd - the password to identify user by
 ynh_psql_create_db() {
 	local db=$1
+	local user=$2
```

Comment:
If user is an optional argument, it should have a default value

Reply:
I can move this line after the `if [ $# -gt 1 ]; then`, or set a default argument. What do you think about it?

Reply:
Sorry, miss read, I though it was a helper with getopts.
Not sure though TRAP would not trig an empty $2. Did you try it ?
Otherwise, you can simply put `local user=${2:-}` to prevent a failure.

Reply:
You're right, this is a problem when we don't use this optional argument.